requestStorageAccessForOrigin.bs

```diff
@@ -221,8 +222,43 @@ Modify the [=determine the storage access policy=] algorithm by prepending the f
+
+As with [=requestStorageAccess=], the same tension between user consent and prompt fatigue exists with {{Document/requestStorageAccessForOrigin(origin)}}; much like the Storage Access API, 
+[=implementation-defined=] acceptance and rejection steps are intended to enable implementers with differing stances on this question to make compromises as they see fit.
 
 <h2 id="security">Security considerations</h2>
 
-ISSUE: Write this section.
+It is important that {{Document/requestStorageAccessForOrigin(origin)}} not degrade security properties of the web platform, even when compared to post-removal of cross-site cookies. 
+Third-party cookie removal has potential benefits for security, specifically in mitigating attacks that rely upon authenticated requests, e.g. CSRF. 
+We do not wish {{Document/requestStorageAccessForOrigin(origin)}} to be a foothold for such attacks to leverage.
+
+Note that <a href="https://privacycg.github.io/storage-access/#security">the Storage Access API security</a> properties hold for much of this proposal. Specifically, frame-level access is only granted once [=requestStorageAccess=] is successfully invoked.
+For frame access, {{Document/requestStorageAccessForOrigin(origin)}} merely simplifies the activation and prompting requirements.
+
+{{Document/requestStorageAccessForOrigin(origin)}} does expand the scope of concerns in two areas: subresource requests made by the top-level document and potential notification abuse.
+
+<h3 id="subresources">Subresource Requests</h3>
+
+The specific security controls proposed by the API are:
+* Any cookies included with the subresource request have to be explicitly marked `SameSite=None`, indicating intent for use in [=third party contexts=].
+* For any `SameSite=None` cookies to be included, the request's [=request/mode=] has to be "cors", where reading of the response is blocked unless the embeddee opts-in via sending the appropriate [:http-access-control-allow-credentials:] header. The sending of the [:http-origin:] header ensures the embeddee is aware of the embedder's identity.
+
+Additionally, only requests initiated from the top-level document will be eligible for inclusion of `SameSite=None` cookies. This ensures that other embedded frames do not receive escalated privileges.
+
+<h3 id="notification-abuse">Notification Abuse</h3>
+
+Unlike the Storage Access API, interaction is only required with the top-level document, rather than an embedded document. This does increase the likelihood of prompting.
+
+Like the Storage Access API, user activation is consumed on denial, which prevents repeated requests.
+
+The [=implementation-defined=] rejection steps also allow for imposition of numeric limits or denylists for abusive actors.
+
+As mentioned in privacy considerations, because of the direction of the request, prompt language has to be careful to indicate which entity is requesting access.
```

Comment:
I think you should be able to link to the privacy considerations with `[[#privacy]]`.